test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -19,6 +19,44 @@ private static IEnumerable<TagHelperDescriptor> DefaultPAndInputTagHelperDescrip
         private static IEnumerable<TagHelperDescriptor> PrefixedPAndInputTagHelperDescriptors
             => BuildPAndInputTagHelperDescriptors("THS");
 
+        private static IEnumerable<TagHelperDescriptor> MinimizedTagHelperDescriptors
```

Comment:
No they cannot, it's just descriptors used for the `MinimizedTagHelperDescriptors` test case.